roles/tower/templates/supervisor.yaml.j2

```diff
@@ -0,0 +1,145 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ meta.name }}-supervisor-config
+  namespace: {{ meta.namespace }}
+  labels:
+    app: tower
+data:
+  supervisor-web-config: |
+    [supervisord]
```

Comment:
Isn't all this config in the image itself? Why does this need to be the responsibility of the operator? (When I'm inspecting the AWX images, at least, I see supervisord setting itself up correctly)

Reply:
This is something we're actively working on fixing. The problem is downstream only... In 3.7.0, we this moved out of the image built in Red Hat's infrastructure. This was a bad idea, and will be fixed in an upcoming 3.7.x release. But for now, we need this because we need to override this: https://github.com/geerlingguy/tower-operator/pull/45/files#diff-858c0f683bcf356ee3c28b5d41dd2daeR43

Note, this will break AWX... I need to fix this up before merging this.